### PR TITLE
Freenode -> Libera.Chat

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
https://github.com/ansible-community/community-topics/issues/19

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`.github/workflows/ansible-test.yml`

### CODE REVIEW

**Review by briantist — APPROVED** [review]
This looks good to me. Only thing I might add is that the channel has `+r` so registration is required to join the channel (this wasn't the case on Freenode until the other day when it got spammed).

Even though it might be considered out of scope, linking to registration instructions would probably go a long way with users.

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # README FIRST
 # 1. If you don't have unit tests remove that section
 # 2. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
-# If you need help please ask in #ansible-devel on Freenode IRC
+# If you need help please ask in the #ansible-community channel on Libera.Chat IRC (https://libera.chat/)
```
Comment (Andersson007) [inline .github/workflows/ansible-test.yml:4]:
```suggestion
# If you need help please ask in the #ansible-community channel on Libera.Chat IRC (https://libera.chat/). Refer to https://libera.chat/guides/registration to learn how to register.
```

**Comment by Andersson007** [comment]
@briantist thanks for the feedback! I added a sentence, PTAL

**Comment by briantist** [comment]
@Andersson007 thanks, to give you an idea where my head's at, I didn't realize that 1) this PR was in my collection 🤦‍♂️ (I clicked via email) and 2) that it's updating a _comment_ inside the GHA workflow, something not really for end users anyway. Sorry to trigger another workflow just for that haha, I'll merge once that completes, just for completeness 😅.

Thanks again for updating this!

**Comment by Andersson007** [comment]
@briantist hehe, I also thought that it's strange a bit but decided that it'd be faster to add than discuss:) Anyway it's not worse now:) Thanks!

**Pull request merged**